CONTRIBUTING.md

```diff
@@ -5,3 +5,5 @@ When contributing, please indicate your data sources.
 Make sure the tests pass and submit a Pull Request.
 
 It is possible to visualize and edit the data with https://trainline-eu.github.io/stations-studio. In order to make pull requests, you must be member of Trainline EU’s organisation. However it is possible to download the modified file without creating a pull request.
+
+Please note: Changes to station names here not only impact what is shown in the search widget and booking flow, but also the SEO landing page URLs. For more info please see [this wiki](https://wiki.thetrainline.com/display/VUL/SEO+Station+Slugs+solution) on how the "SEO slugs" logic is implemented and the impact of updating a station name.
```

Comment:
I think we should incorporate the change, however it would be best to phrase it in an agnostic way without mentionning internal sources.